11_taxonomic_information.Rmd

```diff
@@ -182,7 +182,8 @@ For complete list of available transformations and parameters, see function
 [help](https://microbiome.github.io/mia/reference/transformCounts.html).
 
 ```{r}
-tse <- transformSamples(x = tse, abund_values = "counts", method = "clr", 
+tse <- transformSamples(tse, method = "relabundance")
+tse <- transformSamples(x = tse, abund_values = "relabundance", method = "clr", 
                         pseudocount = 1, name = "clr_transformation")
```

Comment:
1 They should give equal result, but we should probably pick which one we use: pseudocount in relabundance step or in clr step